Documentation/project-docs/performance-tests.md

```diff
@@ -44,7 +44,10 @@ For the time being, perf tests should reside within their own "Performance" fold
 Start by adding the following lines to the tests csproj:
 ```
  <!-- Performance Tests -->  
-  <ItemGroup Condition="'$(RunPerfTestsForProject)' == 'true'">  
+  <PropertyGroup Condition="'$(Performance)' == 'true'">
+    <IncludePerformanceTests>true</IncludePerformanceTests>
+  </PropertyGroup>
```

Comment:
Rather than doing this mapping in every .csproj, should we add it to https://github.com/dotnet/corefx/blob/master/dir.props?